### ISSUE TITLE
fbvnc only show 'showmsg()' from fbnvc.c

### ISSUE THREAD

**Issue by tomnormbgu** [aligrudi/fbvnc#5]
Hi,
I'm using alpine linux 3.20. Im using Xvnc and Qemu's vnc, in both when I open fbvnc from the TTY I get a black screen with the output of showmsg() from fbvnc.c.
when I press ctrl+space several times I get (discrete) updates to the screen, so I tried to add OUT("1") to fb_set(), and that gives me the vnc screen (but '1' is written all over it when I move the cursor or press the keyboard).
I figured that my framebuffer is updated when I write to it (like in OUT) but not when I memcpy to it (like in fb_set).

The output of `fbset` is:

mode "1920x1080-0"
	# D: 0.000 MHz, H: 0.000 kHz, V: 0.000 Hz
	geometry 1920 1080 1920 1080 32
	timings 0 0 0 0 0 0 0
	accel false
	rgba 8/16,8/8,8/0,0/0
endmode

Do you have an idea what is the problem?

Thanks!

**Comment by aligrudi** [comment]
tomnormbgu ***@***.***> wrote:
> I'm using alpine linux 3.20. Im using Xvnc and Qemu's vnc, in both when I open fbvnc from the TTY I get a black screen with the output of showmsg() from fbvnc.c.
> when I press ctrl+space several times I get (discrete) updates to the screen, so I tried to add OUT("1") to fb_set(), and that gives me the vnc screen (but '1' is written all over it when I move the cursor or press the keyboard).
> I figured that my framebuffer is updated when I write to it (like in OUT) but not when I memcpy to it (like in fb_set).
> 
> The output of `fbset` is:
> 
> mode "1920x1080-0"
> 	# D: 0.000 MHz, H: 0.000 kHz, V: 0.000 Hz
> 	geometry 1920 1080 1920 1080 32
> 	timings 0 0 0 0 0 0 0
> 	accel false
> 	rgba 8/16,8/8,8/0,0/0
> endmode
> 
> Do you have an idea what is the problem?

Are you using X11 or Wayland at the same time?

	Ali

**Comment by tomnormbgu** [comment]
no. one of them at a time. for now I'm only using X11 via Xvnc.
